parquet-protobuf/src/main/java/org/apache/parquet/proto/ProtoWriteSupport.java

```diff
@@ -318,7 +318,7 @@ final void writeRawValue(Object value) {
     @Override
     final void writeRawValue(Object value) {
       ByteString byteString = (ByteString) value;
-      Binary binary = Binary.fromByteArray(byteString.toByteArray());
+      Binary binary = Binary.fromConstantByteArray(byteString.toByteArray());
```

Comment:
I do not think it is reused right now. But, the fact that producer can potentially change it makes "reused" version suitable to be used here.